x-pack/plugins/index_management/__jest__/client_integration/home.test.ts

```diff
@@ -514,5 +514,70 @@ describe('<IndexManagementHome />', () => {
         });
       });
     });
+
+    describe('index detail panel with % character in index name', () => {
+      const indexName = 'test%';
+      beforeEach(async () => {
+        const index = {
+          health: 'green',
+          status: 'open',
+          primary: 1,
+          replica: 1,
+          documents: 10000,
+          documents_deleted: 100,
+          size: '156kb',
+          primary_size: '156kb',
+          name: indexName,
+        };
+        httpRequestsMockHelpers.setLoadIndicesResponse([index]);
+
+        const { find, component } = testBed;
+
+        await act(async () => {
+          find('reloadIndicesButton').simulate('click');
+          await nextTick();
+          component.update();
```

Comment:
It would be better to remove the `nextTick()` call and put the `component.update()` outside the `act()` call 👍

Reply:
@sebelga Thanks a lot for the tip, updated the tests with your suggestions!